### PR TITLE
First version of ITSMFT Cage: cover sheets and cover ribs

### PR DESCRIPTION
**Pull request by mario6829**
A few elements of the ITS-MFT support cage have been implemented in the geometry: the upper and lower cover sheets and cover ribs. Volume description  is based on latest blueprints. These elements are placed as single nodes in the "barrel" volume.

### CODE REVIEW

File: Detectors/ITSMFT/ITS/simulation/src/V3Cage.cxx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+  TGeoVolume* coverVol = new TGeoVolume("CageCover", coverSh, medPrepreg);
+  coverVol->SetFillColor(kBlue);
+  coverVol->SetLineColor(kBlue);
+
+  TGeoVolume* coverCoreVol = new TGeoVolume("CageCoverCore", coreSh, medRohacell);
+  coverCoreVol->SetFillColor(kYellow);
+  coverCoreVol->SetLineColor(kYellow);
+
+  coverVol->AddNode(coverCoreVol, 1, 0);
```
Comment (mconcas) [inline Detectors/ITSMFT/ITS/simulation/src/V3Cage.cxx]: Hi @mario6829, this line makes fullCI failing.
```/sw/SOURCES/O2/8693-slc8_x86-64/0/Detectors/ITSMFT/ITS/simulation/src/V3Cage.cxx:207:38: error: use nullptr [modernize-use-nullptr]```

can you change the 0 with a `nullptr`?

**Review by iouribelikov — DISMISSED** [review]
It seems to work fine for me.

**Comment by sawenzel** [comment]
pinging @amorsch to give chance to comment

File: Detectors/ITSMFT/ITS/simulation/src/V3Cage.cxx (hunk 1 above)
Comment (mario6829) [reply Detectors/ITSMFT/ITS/simulation/src/V3Cage.cxx]: Hi @mconcas thanks for pointing it out! Fixed

**Comment by amorsch** [comment]
Got confirmation that implementation has been coordinated with MFT.
Ok for me.

**Review by mconcas — APPROVED** [review]

**Review by rpezzi — APPROVED** [review]

**Pull request merged**